src/storage/v3/conversions.hpp

```diff
@@ -0,0 +1,127 @@
+// Copyright 2022 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#include "expr/typed_value.hpp"
+#include "storage/v3/property_value.hpp"
+
+#pragma once
+
+namespace memgraph::storage::v3 {
+
+template <typename TypedValue>
+TypedValue PropertyToTypedValue(const PropertyValue &value) {
+  switch (value.type()) {
+    case storage::v3::PropertyValue::Type::Null:
+      return TypedValue();
+    case storage::v3::PropertyValue::Type::Bool:
+      return TypedValue(value.ValueBool());
+    case storage::v3::PropertyValue::Type::Int:
+      return TypedValue(value.ValueInt());
+    case storage::v3::PropertyValue::Type::Double:
+      return TypedValue(value.ValueDouble());
+    case storage::v3::PropertyValue::Type::String:
+      return TypedValue(value.ValueString());
+    case storage::v3::PropertyValue::Type::List: {
+      const auto &src = value.ValueList();
+      std::vector<TypedValue> dst;
+      dst.reserve(src.size());
+      // std::transform(src.begin(), src.end(), std::back_inserter(dst), [](const auto& elem) { return
+      // PropertyToTypedValue(elem); });
+      for (const auto &elem : src) {
+        dst.push_back(PropertyToTypedValue<TypedValue>(elem));
+        //      std::transform(src.begin(), src.end(), std::inserter(dst, dst.end()), [](const auto &pr) {
+        //        return std::pair(pr.first, TypedToPropertyValue(pr.second));
+        //      });
```

Comment:
What about the commented out code parts?

Reply:
removed :)